src/commands/edge_app_server.rs

```diff
@@ -221,12 +288,8 @@ settings:
             .await
             .unwrap();
         let content = resp.text().await.unwrap();
-
-        assert!(content.contains("screenly ="));
-        assert!(content.contains("http://127.0.0.1:8080")); // Assuming this is expected from your server
-        assert!(content.contains("metadata: {")); // Make sure metadata is in its own section
-        assert!(content.contains("\"srly-t6kb0ta1jrd9o0w\"")); // Checking an example metadata value
-        assert!(content.contains("key"));
+        let expected_content = "var screenly = {\n    metadata: {\n        \"coordinates\": [\"37.3861\", \"-122.0839\"],\n        \"hardware\": \"x86\",\n        \"hostname\": \"srly-t6kb0ta1jrd9o0w\",\n        \"location\": \"Silicon Valley, USA\",\n        \"screen_name\": \"Code Cafe Display\",\n        \"tags\": [\"All Screens\"]\n    },\n    settings: {\n        \"enable_analytics\": \"true\",\n        \"override_timezone\": \"\",\n        \"tag_manager_id\": \"\"\n    },\n    secrets: {\n        \"key\": \"value\"\n    },\n    cors_proxy_url: \"http://127.0.0.1:8080\"\n};";
```

Comment:
maybe format it as multiline string to make it more readable?

Reply:
Done!